backend/src/main/java/com/zeros/backend/service/Route.java

```diff
@@ -0,0 +1,64 @@
+package com.zeros.backend.service;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+import java.util.Objects;
+
+public class Route  {
```

Comment:
Should we call it RouteDistance or something else more expressive than Route. From my perspective Route stands more for an itinerary and in our model it's not really the itinerary than we are modeling

Reply:
@zeroual, still a Route is more like a path from point A to B, we can imagine it as a list of locations for instance, however it's not so confusion we can think of it later.